src/mono/mono/metadata/assembly-internals.h

```diff
@@ -138,6 +138,9 @@ mono_assembly_get_name_internal (MonoAssembly *assembly);
 MONO_PROFILER_API MonoImage*
 mono_assembly_get_image_internal (MonoAssembly *assembly);
 
+MonoBundledAssembly *
```

Comment:
Is this still needed, didn't find call in this PR. Since we move away from MonoBundledAssembly, maybe there is no need to add more function dealing with it?

Reply:
Good catch, overlooked that this was still added.